### PR TITLE
Import css dropdowns handling from udata-gouvfr

### PR DESCRIPTION
**Pull request by noirbizarre**
This PRs import CSS dropdowns behavior from gouvfr style (it's a ashred behavior) and fix hidden submenus on mobile.

screenshot-www data dev-2017-12-11-12-02-36-033

### CODE REVIEW

File: less/udata/dropdowns.less [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+        display: block;
+    }
+
+    // Clickable dropdowns
+    &.open .dropdown-menu {
+        display: inline;
+    }
+
+    // Mobile: display inline when menu is open
+    .navbar-collapse.in &.dropdown-hover .dropdown-menu {
```
Comment (oncletom) [inline less/udata/dropdowns.less]: How do we know this is for mobile? Just by reading the CSS selector, it is not obvious.

Comment (noirbizarre) [reply less/udata/dropdowns.less]: `.navbar-collapse.in` is the bootstrap selector for the expanded menu on mobile only (`.navbar-collapse` is always visible on desktop)
This is not obvious (and I had to search), this is why I put the comment.

Comment (noirbizarre) [reply less/udata/dropdowns.less]: BTW, why block the merge and request change ?

Comment (oncletom) [reply less/udata/dropdowns.less]: @noirbizarre ok thanks. I was not sure if there was a more obvious way to signpost this. Something more obvious than a comment. I can live with it :-)

Comment (noirbizarre) [reply less/udata/dropdowns.less]: Some mecanism in bootstrap are a little bit weird to understand :/

**Pull request merged**